app/controllers/responses_controller.rb

```diff
@@ -13,6 +13,11 @@ def create
     @response.longitude = coords[:longitude]
     if @response.save
       ResponseMailer.with(response: @response, chomp_session: @chomp_session).create_response.deliver_later if user_signed_in?
+      # check invitees limit, check if user inputted any user-limit and if responses count already exceeded
+      if @chomp_session.invitees != nil && @chomp_session.responses.count == (@chomp_session.invitees + 1)
+        GenerateRestaurantJob.perform_now(@chomp_session.id)
```

Comment:
if there's a way to get the scheduled job to perform now, that would be even better!